fvtest/compilertest/build/toolcfg/gnu/common.mk

```diff
@@ -143,13 +143,13 @@ endif
 
 ifeq ($(HOST_ARCH),z)
     ifeq ($(HOST_BITS),32)
```

Comment:
OpenJ9 defines `HOST_BITS` as 31 or 64 for z/OS, perhaps this should be `ifneq $(HOST_BITS),64)`?

Reply:
Fixed in 370fd8ea